### PR TITLE
chore: basic structure for multi-d feature-lib

### PR DESCRIPTION
**Pull request by psmul**
Closes: #10742

### CODE REVIEW

**Review by dydome — APPROVED** [review]

File: feature-libs/product/multi-d/components/variants-multi-dimensional-selector/variants-multi-dimensional-selector.component.ts [hunk 1]
```diff
@@ -34,11 +34,13 @@ export class VariantGenericSelectorComponent implements OnInit {
   }
 
   changeVariant(code: string): void {
+    console.log('changeVariant', code);
```
Comment (dydome) [inline feature-libs/product/multi-d/components/variants-multi-dimensional-selector/variants-multi-dimensional-selector.component.ts]:
```suggestion
    
```

File: feature-libs/product/multi-d/components/variants-multi-dimensional/variants-multi-dimensional.component.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+})
+export class VariantsMultiDimensionalComponent implements OnInit {
+  product$: Observable<Product | null>;
+
+  constructor(private currentProductService: CurrentProductService) {}
+
+  ngOnInit(): void {
+    this.product$ = this.currentProductService.getProduct().pipe(
+      filter((product) => {
+        console.log(product);
```
Comment (dydome) [inline feature-libs/product/multi-d/components/variants-multi-dimensional/variants-multi-dimensional.component.ts]:
```suggestion
```

File: feature-libs/product/multi-d/components/order-grid/order-grid.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+Order grid works
```
Comment (psmul) [inline feature-libs/product/multi-d/components/order-grid/order-grid.component.html:1]: Can we change it so it contains proper html code?

File: feature-libs/product/multi-d/components/order-grid/order-grid.component.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+  selector: 'cx-order-grid',
+  templateUrl: './order-grid.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OrderGridComponent implements OnInit {
```
Comment (psmul) [inline feature-libs/product/multi-d/components/order-grid/order-grid.component.ts:8]: I believe this should be something more contextual, related to variants

**Pull request merged**